app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="login_form">
+  <h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field remember">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=Modify%20the%20views%20in%20devise%20for%20registration%20and%20login%20to%20match%20the%20wireframes%20from%20the%20Sneak%20Peek%20and%20your%20styling.), Please modify the styling of the sign-in page to match the given wireframe design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/forLoop94/blogApp/assets/68796232/14f64340-226c-4848-badd-85a35b1616ae)  | ![image](https://github.com/forLoop94/blogApp/assets/68796232/373a39db-9828-4fe7-ae1c-76d5b71c16a8) |